doc/xml/user-guide.xml

```diff
@@ -2006,10 +2011,10 @@
             </execute>
         </execute-list>
 
-        <p>The default behavior for restore is to use the last backup but an earlier backup can be specified with the <br-option>{[dash]}-set</br-option> option.</p>
+        <p>The default behavior for time-based restore, if the <br-option>{[dash]}-set</br-option> option is not set, is to attempt to discover an earlier backup to play forward from; if a backup set cannot be found, then restore will default to the latest backup which, as shown earlier, may not give the desired result.</p>
 
         <execute-list host="{[host-pg1]}">
-            <title>Stop <postgres/>, restore from the selected backup, and start <postgres/></title>
+            <title>Stop <postgres/>, allow <backrest/> to discover the backup, and start <postgres/></title>
```

Comment:
This title should still mention that a restore is happening, maybe "restore from auto-selected backup" or "restore from discovered backup".  Maybe auto-select is generally a better term than discover?

Reply:
Sounds good